analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    census['area(sq_miles)']= census.apply(lambda x: area_sq_miles(x.geometry), axis=1)
</pre><p>nit: we can just call this 'area'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Sure. I will mention in a comment that the area is in square miles. I will make these changes in the other notebook - <a href="https://github.com/priyakalyan/Updated_NC_pop/blob/main/NC_updated_pop_comparison.ipynb" target="_blank">NC_updated_pop_comparison.ipynb</a> too. </p>